Makefile

```diff
@@ -9,7 +9,7 @@ MIN_FILE_SIZE = 1000
 
 # Start the server
 s:
-	REDIS_URL=redis://127.0.0.1:6379 API_URL=http://localhost:3000/v2 APP_URL=http://localhost:5000 foreman start
+	REDIS_URL=redis://127.0.0.1:6379 API_URL=http://localhost:3000/v2 APP_URL=http://localhost:5000 GRAPHQL_ENDPOINT=http://localhost:3000/graphql foreman start
```

Comment:
y'all needa start using envs

Reply:
I do, I just wanna have a make command set it up for me

Reply:
![](http://static.damonzucconi.com/_capture/1xqcX0Q8s1.png)